lib/common/adjust_zone.js

```diff
@@ -1,17 +1,16 @@
 var adjust_lon = require('./adjust_lon');
 
 module.exports = function(zone, lon) {
-  if (!zone) {
+  if (zone === undefined) {
     zone = Math.floor((adjust_lon(lon) + Math.PI) * 30 / Math.PI);
 
     if (zone < 0) {
       return 0;
-    }
-    else if (zone >= 60) {
+    } else if (zone >= 60) {
       return 59;
     }
-  }
-  else {
+    return zone;
```

Comment:
The `mapshaper-proj` works with just `zone` here, and I took the code from there, basically, so it should work with `zone`.